readme.md

```diff
@@ -81,7 +82,22 @@ Your Rails database has two tables. `students` has the columns `id` and `name`,
 Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named Geraldo.
 
 ```rb
-# Your answer...
+# Normally the ID would be generated for you but it's specifically asked in the question so i'm putting it in
+
+ActiveRecord::Migration
+create_table :students do |t|
+  t.string :id
+  t.string :name
+end
+
+create_table :sandwhices do |t|
+  t.string :id
+  t.string :type
+  t.string :student_id
+
+Studnet.students create (id: 1, name: "Geraldo")
+Student.sandwhiches create(id:1 , type: "pbj", student_id: 1)
+
```

Comment:
Close.
- Let ActiveRecord manage the id column.  It will create it so you can use it.  The migration you wrote will create the primary key and foreign key as strings, instead of integers.
- The spaces between the method call and the arguments will break ruby.
- We don't sign id's in AR.
- Last line should use `geraldo`
```
geraldo.sandwiches.create!(type: "pbj")
```
This will add a new sandwich for Geraldo.
Bonus: "type" is a reserved column name in Rails.  We should avoid it.